### ISSUE TITLE
Groups.py output empty

### ISSUE THREAD

**Issue by tulinkry** [oracle/opengrok#2372]
I expect the output of these two commands to be the same, did I miss something when using the `groups.py` command?

```bash
-bash-4.1$ /opt/python/3.5/bin/python opengrok/bin/groups.py -j /usr/java/jdk1.8.0/bin/java --jar opengrok/lib/opengrok.jar -D -- -n org -r ahoj
DEBUG:groups.py:Java = /usr/java/jdk1.8.0/bin/java
DEBUG:groups.py:Java command: ['/usr/java/jdk1.8.0/bin/java', '-classpath', 'opengrok/lib/opengrok.jar', 'org.opengrok.indexer.configuration.Groups', '-n', 'org', '-r', 'ahoj']
DEBUG:groups.py:working directory = /data/5e3a0f6e-282c-4245-a345-61b514b1254d/app/opengrok
DEBUG:groups.py:command = ['/usr/java/jdk1.8.0/bin/java', '-classpath', 'opengrok/lib/opengrok.jar', 'org.opengrok.indexer.configuration.Groups', '-n', 'org', '-r', 'ahoj']
DEBUG:groups.py:Waiting for process with PID 27895
DEBUG:groups.py:done waiting
DEBUG:groups.py:['/usr/java/jdk1.8.0/bin/java', '-classpath', 'opengrok/lib/opengrok.jar', 'org.opengrok.indexer.configuration.Groups', '-n', 'org', '-r', 'ahoj'] -> 0
DEBUG:groups.py:closed
DEBUG:groups.py:Waiting on output thread to finish reading
DEBUG:groups.py:end of output
DEBUG:groups.py:Command ['/usr/java/jdk1.8.0/bin/java', '-classpath', 'opengrok/lib/opengrok.jar', 'org.opengrok.indexer.configuration.Groups', '-n', 'org', '-r', 'ahoj'] took 0 seconds
```

This is the command from above run directly:

```bash
-bash-4.1$ /usr/java/jdk1.8.0/bin/java -classpath opengrok/lib/opengrok.jar org.opengrok.indexer.configuration.Groups -n org -r ahoj
<?xml version="1.0" encoding="UTF-8"?>
<java version="1.8.0" class="java.beans.XMLDecoder">
 <object class="org.opengrok.indexer.configuration.Configuration" id="Configuration0">
  <void property="groups">
   <void method="add">
    <object class="org.opengrok.indexer.configuration.Group">
     <void property="name">
      <string>org</string>
     </void>
     <void property="pattern">
      <string>ahoj</string>
     </void>
    </object>
   </void>
  </void>
 </object>
</java>
-bash-4.1$
```

**Comment by vladak** [comment]
dup of #2356 ?

**Comment by vladak** [comment]
Should be fixed in 1.1-rc42.

**Comment by tulinkry** [comment]
Aha. Yes I guess so.
